lib/components/grpc.ts

```diff
@@ -771,10 +771,25 @@ export default function createGrpcAction<Context extends GatewayContext>(
             'x-request-id': requestId,
             'x-gateway-version': VERSION,
         };
+
         if ('protoPath' in config) {
             debugHeaders['x-api-request-protopath'] = config.protoPath;
         }
 
+        if (headers['uber-trace-id']) {
+            debugHeaders['x-uber-trace-id'] = headers['uber-trace-id'];
+        }
+
+        if (typeof options.proxyDebugHeaders === 'function') {
+            Object.assign(debugHeaders, options.proxyDebugHeaders({...headers}, 'grpc'));
+        } else if (Array.isArray(options.proxyDebugHeaders)) {
+            for (const headerName of options.proxyDebugHeaders) {
+                if (headers[headerName] !== undefined) {
+                    debugHeaders[`x-${headerName}`] = headers[headerName];
```

Comment:
I think it's not best idea to add `x-` prefix in itself because it can lead to headers like `x-x-request-id`, `x-x-forwarded-for` and others, which is not entirely clear. I suggest to use at least `x-header-` prefix to separate them from default debug headers. But generally it's not a big deal

Reply:
I've decided to use `x-gateway` prefix